### PR TITLE
General refactoring and added nonechucks support.

### PR DESCRIPTION
**Pull request by afiaka87**
Unfortunately nonechucks is no longer maintained and requires a patch to run on libraries above pytorch=1.3. I can't verify it works but it doesn't break immediately at least.

### CODE REVIEW

**Comment by afiaka87** [comment]
Only way I could get it to run was by forking it. So @lucidrains I'll probably just remove nonechucks support and go back to returning a random int based on `len(dataset)`.

**Comment by afiaka87** [comment]
Contains fix for https://github.com/lucidrains/DALLE-pytorch/issues/161 ( think? this fixes it for me anyway.)

the lowering epsilon helps a _little_ with this issue https://github.com/lucidrains/DALLE-pytorch/issues/169 but it still happens. 

also contrains fix for https://github.com/lucidrains/DALLE-pytorch/issues/186

**Comment by afiaka87** [comment]
also @lucidrains do you mind if we up the default RandomResize? The 0.6 zoom has given me and a few others some poor reconstructions. I think it's because we're training on 256 px images though

**Comment by lucidrains** [comment]
> also @lucidrains do you mind if we up the default RandomResize? The 0.6 zoom has given me and a few others some poor reconstructions. I think it's because we're training on 256 px images though

ohh sure, let me fix that now

File: train_dalle.py [hunk 1]
```diff
@@ -263,67 +259,63 @@ def __getitem__(self, ind):
...
 
-    model_config = dict(
-        depth = DEPTH,
-        heads = HEADS,
-        dim_head = DIM_HEAD
-    )
+    model_config = dict(depth=DEPTH, heads=HEADS, dim_head=DIM_HEAD)
 
+    project_name = "dalle_train_transformer"
+    if args.name not None:
```
Comment (janEbert) [inline train_dalle.py:287]: Missing an `is`, so `not` → `is not`.

**Pull request closed without merging**